astropy/nddata/nddata.py

```diff
@@ -37,20 +30,15 @@ class NDData(NDDataBase):
     Parameters
     -----------
     data : `~numpy.ndarray`, `~numpy.ndarray`-like, or `NDData`
-        The actual data contained in this `NDData` object. If possible, data
-        will not be copied`data`, so you should make copy the ``data`` before
-        passing it in if that's the desired behavior.
+        The actual data contained in this `NDData` object.
 
     uncertainty : any type, optional
-        Uncertainty on the data. The uncertainty *must* have a string attribute
-        named ``uncertainty_type``, but there is otherwise no restriction.
+        Uncertainty on the data. The uncertainty *should* have a string
```

Comment:
I'm think we'll need to stick with `*must*` -- this point got a ton of discussion in APE 7....